server/grpc_ac.go

```diff
@@ -44,10 +44,14 @@ func (s *grpcServer) GetActionResult(ctx context.Context,
 		return nil, err
 	}
 
+	// Clients provides hash and size of the Action, but not size of the ActionResult
+	// checked by the the disk cache.
+	const sizeActionResult = -1
```

Comment:
Maybe rename this to `unknownActionResultSize`?

Reply:
Done